Sources/TootSDK/Models/DomainBlock.swift

```diff
@@ -18,7 +18,7 @@ public enum DomainBlockSeverity: String, Codable, Hashable {
 }
 
 /// Represents a domain limited from federating.
-public struct DomainBlock: Codable, Hashable {
+public struct DomainBlock: Codable, Hashable, Identifiable {
```

Comment:
This one I would keep as not `identifiable` and even make the `id` optional as it's not provided supported by all servers.